pySDC/projects/DAE/problems/simple_DAE.py

```diff
@@ -49,6 +55,7 @@ def __init__(self, nvars, newton_tol):
         # solution = data[:, 1:]
         # self.u_ref = interp1d(t, solution, kind='cubic', axis=0, fill_value='extrapolate')
         self.t_end = 0.0
+        self.work_counters['rhs'] = WorkCounter()
```

Comment:
Looks like at least this one could go to the generic parent class, too?

Reply:
True! I added this for all problem classes. I'll do that.

Reply:
Looks better and more convenient (like before), thanks a lot!